.ibm/pipelines/kubernetes-tests.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+LOGFILE="pr-${GIT_PR_NUMBER}-kubernetes-tests-${BUILD_NUMBER}"
+
+source .ibm/pipelines/functions.sh
+
+install_ibmcloud cloud-object-storage kubernetes-service
+
+ibmcloud ks cluster config --cluster "${IBM_KUBERNETES_ID}" --admin
+
+install_kubectl
+
+(
+    set -e
+    make install
+    make test-integration-devfile
+    make test-e2e-devfile
+    make test-cmd-project
+) |& tee "/tmp/${LOGFILE}"
+RESULT=${PIPESTATUS[0]}
+
+install_gh
+save_logs "${LOGFILE}" "Kubernetes Tests"
```

Comment:
@feloy this save log is added as a comment on gihub by itself or do we need to enable anything on pipleline side

Reply:
All is done here, there is nothing else to be done (except getting a private key to connect to GitHub)